ansible_collections/arista/avd/roles/eos_validate_state/tasks/anta_tests.yml

```diff
@@ -18,3 +18,26 @@
   check_mode: false
   when: use_anta | bool
```

Comment:
so stupid question but do we still need the `when`, now that this can only be included when `use_anta` is true in the first place? :)

Reply:
Agreed. Fixed in 359bec3b2eed78e105c6eb215a979e7a773bc325